app/views/cases/cover_pages/show.html.slim

```diff
@@ -12,6 +12,8 @@ article.cover-sheet
         = @case.subject_full_name&.upcase
       li
         = @case.first_prison_number&.upcase
+      li
+        = @case.other_subject_ids&.upcase 
```

Comment:
@willmcb , not fully sure this change meets the criteria `AC2` in the ticket, you may have a look at it again

Reply:
Yeah sorry I read read and will adapt it - apologies for wasting your time 😞